README.md

```diff
@@ -234,6 +200,22 @@ To build a new release package for foreman project for example, do this:
 
     $ tito release koji-foreman
 
+## Releasing a package
+
+After a PR has been merged to add or update a package, then the package should be released.  In case it's a new package, be sure to add it to the correct koji tags as well.  [Here's a helper script](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rel-eng/compare-with-tags.rb)
+
+1. Switch to the directory for the package to be released, i.e. for the rubygem-apipie-rails package:
+
+    `$ cd packages/foreman/rubygem-mysql2/`
+
+1. Do a scratch build of the release first:
+
+    `$ tito release --scratch koji-foreman koji-foreman-plugins`
+
+1. Verify package built successfully in koji, and then release:
+
+    `$ tito release koji-foreman koji-foreman-plugins`
+
```

Comment:
I'd add a note here that it uses local files so be sure to not have local changes.